microsoft-edge/webview2/concepts/enterprise.md

```diff
@@ -28,6 +28,18 @@ IT admins can use group policy objects (GPO) to configure policy settings for We
 
 
 <!-- ====================================================================== -->
+## Windows Server Update Services (WSUS)
+
+[Windows Server Update Services (WSUS)](/windows-server/administration/windows-server-update-services/get-started/windows-server-update-services-wsus) enables information technology administrators to deploy the latest Microsoft product updates. You can use WSUS to fully manage the distribution of updates of WebView2 that are released through Microsoft Update to computers on your network.
+
+> [!NOTE]
+> The recommended way of receiving WebView2 updates is by using the default Edge updater. Any modification of update & servicing paths should be done with caution.
```

Comment:
```suggestion
> The recommended way of receiving WebView2 updates is by using the default Microsoft Edge updater. Any modification of update & servicing paths should be done with caution.
```

Reply:
Also, should "Microsoft Edge updater" be a link? Is that the official name? I couldn't find a good link ...

Reply:
```suggestion
> The recommended way of receiving WebView2 updates is by using the default Edge updater. Any modification of update and servicing paths should be done with caution.
```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand